portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,13 +40,15 @@
     public long timestamp;
     public String name;
     public int likes;
+    public String email;
 
-    public Task(long id, String title, long timestamp, String name) {
+    public Task(long id, String title, long timestamp, String name, String email) {
```

Comment:
Naming this Task seems to be confusing. I don't understand why it is a task.   Task implies that some action should be taken.  But this class is just a data holder.  Maybe CommentEntry. Or just Entry is a better name?

Either way, part of the confusion is due to the lack of a class comment.